src/icalendar/attr.py

```diff
@@ -1763,13 +1829,157 @@ def set_end_with_locking(
         raise ValueError(msg)
 
 
+def _get_images(self: Component) -> list[Image]:
+    """IMAGE specifies an image associated with the calendar or a calendar component.
+
+    Description:
+        This property specifies an image for an iCalendar
+        object or a calendar component via a URI or directly with inline
+        data that can be used by calendar user agents when presenting the
+        calendar data to a user.  Multiple properties MAY be used to
+        specify alternative sets of images with, for example, varying
+        media subtypes, resolutions, or sizes.  When multiple properties
+        are present, calendar user agents SHOULD display only one of them,
+        picking one that provides the most appropriate image quality, or
+        display none.  The "DISPLAY" parameter is used to indicate the
+        intended display mode for the image.  The "ALTREP" parameter,
+        defined in :rfc:`5545`, can be used to provide a "clickable" image
+        where the URI in the parameter value can be "launched" by a click
+        on the image in the calendar user agent.
+
+    Conformance:
+        This property can be specified multiple times in an
+        iCalendar object or in "VEVENT", "VTODO", or "VJOURNAL" calendar
+        components.
+
+    .. note::
+
+        At the present moment, this property is read-only. If you require a setter,
+        please open an issue or a pull request.
```

Comment:
Based on this note, I think you expect this method and its docstring to be published. However when I alter the docs locally, it still doesn't display. Does it need to be added to `__all__` or something else?

```rst
.. automodule:: icalendar.attr
   :members:
   :private-members:
```

Perhaps these will help?

- https://www.sphinx-doc.org/en/master/usage/extensions/autodoc.html#directive-option-automodule-private-members
- https://www.sphinx-doc.org/en/master/usage/extensions/autodoc.html#directive-autoproperty

Alternatively you could follow the pattern for `icalendar.attr.property_del_duration`, `icalendar.attr.property_get_duration`, and `icalendar.attr.property_set_duration`.

Reply:
Ah, OK, I expected to see its private name of `_get_images`. I didn't understand where it should appear. Noted. Thank you!